src/molecules/dropdown-button/dropdown-button.tsx

```diff
@@ -50,11 +50,11 @@ const DropdownButton: React.FC<DropdownButtonProps> = (
     }
 
     if (size != null) {
-        classNames.push(size);
+        classNames.push(`-${size}`);
```

Comment:
Were the `ButtonSize` and `ButtonStyle` enums changed? Just curious about this - not saying it should change back, if so. I believe we have been trying to move away from coupling the class name to the enum representation

Reply:
No the `ButtonSize` and `ButtonStyle` enums were not changed. They are being used in many places but the issue was all the other places were using `(`-${size}`)` as opposed to `(size)`, and same goes for style. That is also how they are being given styles through the CSS.